playground/src/components/CodingEnvironment/MidenInputs.tsx

```diff
@@ -11,7 +11,7 @@ type MidenInputsProps = {
 const MidenInputs = (props: MidenInputsProps): JSX.Element =>
     <div className="min-w-0 flex-1 box-border">
         <EditorLabel label="inputs" />
-        <Editor height="150px" {...props} />
+        <Editor height="400px" {...props} />
```

Comment:
I think 400px might be too much, since it pushes the large code window quite far down the page on my laptop screen. For most of the simple cases, there isn't that much going on in the inputs, and 400px is more than needed for the outputs. Ideally we would have the option to expand these or keep them smaller, but that's a UI update for the future. I probably wouldn't enlarge it past 250px

Reply:
changed to 250px